go.mod

```diff
@@ -0,0 +1,5 @@
+module server
```

Comment:
the module should be the github repo link

like this: github.com/AliShan302/Task_Schedular